docs/control-structures.rst

```diff
@@ -53,12 +65,40 @@ write::
         }
     }
 
-The names of output parameters can be omitted.
-The output values can also be specified using ``return`` statements,
-which are also capable of :ref:`returning multiple values<multi-return>`.
-Return parameters can be used as any other local variable and they
-are zero-initialized; if they are not explicitly
-set, they stay zero.
+The names of return variables can be omitted.
+Return variables can be used as any other local variable and they
+are zero-initialized. If they are not explicitly
```

Comment:
```suggestion
are initialized with their :ref:`default value <default-value>` and have that value unless explicitly set.
```
I'm not 100% sure about the exact wording, though.

Reply:
@bit-shift Ah yes, this was added elsewhere too 👍